ReCoded-Org/pebble-work-capstone-project
# I121 edit profile functionality

To test this PR:
1. run npm i to install all dependencies. I installed something called hot toast, which makes cute little notifications like this!
![image](https://user-images.githubusercontent.com/17632992/186961579-17e5619f-d955-449b-8271-f58dbfc259b4.png)

2. Make sure you sign in with an account. And then go to profile edit page. Try and upload a profile photo.
![image](https://user-images.githubusercontent.com/17632992/186961625-7af9e735-5edb-414f-bac3-0701291de7db.png)


# Related Issue

- Resolve #121
